### ISSUE TITLE
Alternative authentication

### ISSUE THREAD

**Issue by rhiaro** [dokieli/dokieli#45]
eg. Something with bearer tokens so I can authenticate with (and potentially post to) my own website (that isn't running solid) from dokieli.

**Comment by jeswr** [comment]
It may also worth be investigating adding [FedCM](https://developer.mozilla.org/en-US/docs/Web/API/FedCM_API) to Dokeli.

**Comment by VirginiaBalseiro** [comment]
> It may also worth be investigating adding [FedCM](https://developer.mozilla.org/en-US/docs/Web/API/FedCM_API) to Dokeli.

@jeswr Definitely! The [milestone](https://github.com/dokieli/dokieli/milestone/6) for this work is: 

> Investigate suitable authentication mechanisms and implement one or more of the promising options based on existing or ongoing standards development, e.g., TLS, OIDC, Signing HTTP Messages, FedCM.
